templates/Default/footer.html.twig

```diff
@@ -6,19 +6,19 @@
           <h3>{{ "footer.usefulLinks"|trans({}, "catroweb") }}</h3>
           <ul class="list-vertical">
             <li>
-              <a href="{{ url('catrobat_web_help') }}">{{ "tutorials"|trans({}, "catroweb") }}</a>
+              <a id="help" href="https://catrob.at/ShareHelp">{{ "tutorials"|trans({}, "catroweb") }}</a>
```

Comment:
not sure about this one.... Being on a test server brings you to the share server....